src/main/java/cn/nukkit/Server.java

```diff
@@ -566,6 +566,13 @@ public Level remove(Object key) {
 
         EnumLevel.initLevels();
 
+        for (Map.Entry<Integer, Level> entry : this.getLevels().entrySet()) {
```

Comment:
This code should be in the `Level` class. `Server` is already over populated with random pieces of code.